packages/schematics/angular/ng-new/schema.json

```
@@ -129,9 +129,11 @@
       "x-user-analytics": 14
     },
     "strict": {
-      "description": "Creates a workspace with stricter type checking and stricter bundle budgets settings.",
+      "description": "Creates a workspace with stricter type checking and stricter bundle budgets settings. This will improve maintainability and help catch bugs ahead of time. To learn more, go to https://angular.io/guide/strict-mode",
+      "x-prompt": "Do you want to enforce stricter type checking and stricter bundle budgets in the workspace?\n  This will improve maintainability and help catch bugs ahead of time.\n  To learn more, go to https://angular.io/guide/strict-mode",
```

```suggestion
      "description": "Creates a workspace with stricter type checking and stricter bundle budgets settings. This setting helps improve maintainability and catch bugs ahead of time. For more information, see https://angular.io/guide/strict-mode",
      "x-prompt": "Do you want to enforce stricter type checking and stricter bundle budgets in the workspace?\n  This setting helps improve maintainability and catch bugs ahead of time.\n  For more information, see https://angular.io/guide/strict-mode",
```
Just a couple of changes:
- It's generally better to say things "help improve" instead of "will improve."
- From a style standpoint, we are trying to standardize on the phrase "For more information, see..." when providing a link.

Looks good though!